lib/team.rb

```diff
@@ -0,0 +1,32 @@
+require 'pry'
+
+class Team
+  attr_reader :country,
+              :players,
+              :eliminated
+  def initialize(country, eliminated = false)
+    @country = country
+    @players = []
+    @eliminated = eliminated
+  end
+
+  def eliminated
+    @eliminated = true
+  end
```

Comment:
Close on this! Think you would need an `attr_writer` OR a method called `eliminated=` to make this work in accordance with the spec. Of the two I would probably recommend switching `eliminated` from an `attr_reader` to an `attr_writer`.